extensions/adobe/experience/profile/profile-all.schema.json

```diff
@@ -89,6 +89,26 @@
           }
         },
         "https://ns.adobe.com/experience/profile/consentPoliciesIdMap": {
+          "title": "consent policies map",
+          "meta:status": "deprecated",
+          "type": "object",
+          "meta:xdmType": "map",
+          "additionalProperties": {
+            "title": "consent policy id map",
+            "description": "map of validated fields for a consent policy",
```

Comment:
I would change the title and description to something like:
    "title": "Obsolete consent policy id map",
    "description": "this is not the map you should use. see 'ns.adobe.com/consentPoliciesIdMap"